### ISSUE TITLE
Translation Compatibility

### ISSUE THREAD

**Issue by JulianTillmann** [microsoft/ALAppExtensions#4272]
Hi all,

We have developed a document report extension in AL and have a customer project where we want to use those. The customer needs to have the following languages on those document reports
 
-           English
-           Spanish
-           Czech
-           French
-           Mandarin
-           Italian
-           German
 
We have the following layers for the application:
-           In C/SIDE we did have a problem with different code pages if we import languages in the NAV standard objects as CaptionML. It was previously recommended to use Multilanguage files in this case and we were warned about mixing CaptionML and Multilanguage files.
 
-           We’re using XLIFF files for our document report app, which we can translate so that shouldn’t be a problem.
 
-           The customization app does not use XLIFF files because those are very hard to handle without proper tools. Instead we use CaptionML.
 
So my questions are:
Is it possible to use MultilanguageFiles for C/SIDE / NST, XLIFF for one extension and CaptionML for another or what is the recommended path?
 
And how do we go about getting the Multilanguage files for the aforementioned languages?
 
What about translations in Mandarin for C/SIDE objects of the NAVDACH localization?

**Comment by AlexanderYakunin** [comment]
@JesperSchulz @PredragMaricic

**Comment by sorenfriisalexandersen** [comment]
You either use the *ML properties or XLIFF files, not a combination. If you use *ML property, it will not be translated by TranslationFiles (xliff).

**Comment by JulianTillmann** [comment]
Even when it’s two different apps? @atoader said otherwise at directions emea

**Comment by sorenfriisalexandersen** [comment]
No, the setting is per app. So if you use the TranslationFile flag in the manifest it will decide how the app handles translations.

**Comment by JulianTillmann** [comment]
Yeah, but my question was not regarding one app but two seperate ones and if that would work if one uses xliff and the other captionML

**Comment by sorenfriisalexandersen** [comment]
Let's have @atoader on this
